pages/common/xml-canonic.md

```diff
@@ -0,0 +1,20 @@
+# xml canonic
+
+> Make XML documents canonical.
```

Comment:
Is `canonical` a common term? Other than the company, I've never heard it and it seems quite uncommon.

Reply:
Read the abstract of https://www.w3.org/TR/xml-c14n/ . "Canonic" is the correct term for XML.
BTW "C14N" means "Canonicalization", e.g. 'C', 14 other letters, and 'N'.
Some languages, like Wolfram, refer to the canonical form as "standard form". In mathematics, its "canonical form".